SonarLint.VisualStudio.Integration.sln

```diff
@@ -4,6 +4,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 VisualStudioVersion = 17.0.31815.197
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Integration.Vsix", "src\Integration.Vsix\Integration.Vsix.csproj", "{FF2AD819-28F4-493A-8E9B-1D3F16BD4689}"
+	ProjectSection(ProjectDependencies) = postProject
```

Comment:
I used the `Project Dependencies` dialogue to make sure the exe is built before the Integration.Vsix project. The Integration.Vsix project does not have a `ProjectReference` to the exe project.

![image](https://user-images.githubusercontent.com/10757559/207283049-7195e602-0741-447b-98f7-2b86c44fb888.png)